Example/src/main/java/com/parse/example/MainActivity.java

```diff
@@ -0,0 +1,92 @@
+package com.parse.example;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+
+import com.parse.GetCallback;
+import com.parse.Parse;
+import com.parse.ParseException;
+import com.parse.ParseLiveQueryClient;
+import com.parse.ParseObject;
+import com.parse.ParseQuery;
+import com.parse.SubscriptionHandling;
+import com.parse.interceptors.ParseLogInterceptor;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class MainActivity extends AppCompatActivity {
+
+    String URL = "http://192.168.3.9:1337/parse/";
```

Comment:
Variable named uppercase URL may be confused with the [URL class](https://docs.oracle.com/javase/7/docs/api/java/net/URL.html).
`String url`
I recommend using lowercase to improve readability.